azurerm/internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -90,9 +90,10 @@ func resourceWindowsVirtualMachine() *schema.Resource {
 			"os_disk": virtualMachineOSDiskSchema(),
 
 			"size": {
-				Type:         schema.TypeString,
-				Required:     true,
-				ValidateFunc: validation.StringIsNotEmpty,
+				Type:             schema.TypeString,
+				Required:         true,
+				ValidateFunc:     validation.StringIsNotEmpty,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
(as above) unfortunately this can create issues within Terraform Core, so we should fix the test configurations to use the correct casing instead:

```suggestion
```